src/test/java/uk/gov/ons/ctp/integration/fieldsvccucumber/cucSteps/sso/TestSSOFieldService.java

```diff
@@ -199,6 +144,18 @@ public void the_EQ_launch_event_is_triggered() {
     assertTrue(currentURL.contains("session") && currentURL.contains("token"));
   }
 
+  private String waitForEqLaunch() throws Exception {
+    String url = null;
+    for (int i = 0; i < 30; i++) {
+      url = driver.getCurrentUrl();
+      if (url != null && url.contains(testConfig.getEqHost())) {
+        break;
+      }
+      Thread.sleep(500);
+    }
+    return url;
```

Comment:
I think it should throw an exception if it fails to get the launch page after 15 seconds. 
As it stands it quietly keeps going, leaving no clue as to the failure.

Reply:
done